auto-random.md

```diff
@@ -142,8 +142,9 @@ This attribute supports forward compatibility, namely, downgrade compatibility.
 
```

Comment:
Above, at Line 70:
```
In the above `CREATE TABLE` statement, `3` shard bits are specified. The range of the number of shard bits is `[1, field_max_bits)`. `field_max_bits` is the length of bits occupied by the primary key column.
```

Is probably not fully correct:

```
tidb> create table t (a bigint auto_random(62) primary key, b varchar(255));
ERROR 8216 (HY000): Invalid auto random: max allowed auto_random shard bits is 15, but got 62 on column `a`
```

Reply:
https://github.com/pingcap/tidb/pull/17119 added:

```go
// MaxAutoRandomBits is the max value of auto sharding.
const MaxAutoRandomBits = 15
```

Reply:
5fd5ff499c4e730f9404b79d42642431f8ea7eaf